File: ruby/pangram/pangram.rb
class Pangram
  def self.is_pangram?(string)
    # first downcase, then sort alphabetically, then remove repeated letters
    formatted_string = string.downcase.chars.sort.join.squeeze

    formatted_string.include? 'abcdefghijklmnopqrstuvwxyz'

    #SOLUTION 2: credits to @wayneangoh
     ('a'..'z').all? { |char| str.downcase.include?(char) }
  end
end

File: javascript/pangram/pangram.js
Pangram = function (input) {
  this.input = input
}

Pangram.prototype.isPangram = function () {
  var formattedString = this.input.toLowerCase()
  var answer = [...'abcdefghijklmnopqrstuvwxyz'].every((x) => formattedString.includes(x))
  return answer
}

module.exports = Pangram
